src/components/ResetPasswordForm.jsx

```diff
@@ -98,7 +98,12 @@ const ResetPasswordForm = (props, routeProps) => {
                     : "cs-form-control"
                 }
               >
-                <label htmlFor="Email">{ResetPasswordPage.EmailLabel}</label>
+                {/* <label htmlFor="Email">{ResetPasswordPage.EmailLabel}</label> */}
```

Comment:
Commented code should be removed.

Reply:
I understand but i was not touching code that I was not needed to .

Reply:
if you run into commented code, especially when it's clear it's not needed it can be removed.

Reply:
done